pyartcd/pyartcd/pipelines/update_golang.py

```
@@ -313,6 +342,50 @@ def get_existing_builders(self, el_nvr_map, go_version):
+                        },
+                        extra_patterns=extra_patterns,
+                        limit=1,
+                    ),
+                    None,
+                )
+                for el_v in el_nvr_map
+            )
+        )
+        found_records = {
+            el_v: cast(KonfluxBuildRecord, build_record)
+            for el_v, build_record in zip(el_nvr_map, build_records)
```

Ruff (0.14.2)

`zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)